### ISSUE TITLE
Getting error Unsupported value: <FlutterError: 0x60c000225500> of type FlutterError

### ISSUE THREAD

**Issue by manujbahl** [dart-flitter/flutter_webview_plugin#49]
Do you know what that error means?

**Comment by manujbahl** [comment]
I think it is the http vs https issue. I get that error on iOS when visiting an http link.
Is there a workaround?

**Comment by mikowm** [comment]
Yes, the reason is that you're trying to load a http URL. You can change the .plist file (Info.plist) in your project to add a exception to for specific domains or disable App Transport Security (ATS) to allow all connections. You can fin the code for both solutions [here](https://stackoverflow.com/a/30732693)

**Comment by lejard-h** [comment]
Hi, do you still have this issue ?
